web/client/utils/FilterUtils.jsx

```diff
@@ -530,80 +788,24 @@ const FilterUtils = {
          `</ogc:Function>`;
         }
         if (crossLayerFilter.operation === "DWITHIN") {
-            ogc += '<' + this.nsplaceholder + ':Distance units="m">' + (crossLayerFilter.distance || 0) + '</' + this.nsplaceholder + ':Distance>';
+            ogc += '<' + nsplaceholder + ':Distance units="m">' + (crossLayerFilter.distance || 0) + '</' + nsplaceholder + ':Distance>';
         }
 
-        ogc += this.ogcSpatialOperator[crossLayerFilter.operation].endTag;
+        ogc += ogcSpatialOperator[crossLayerFilter.operation].endTag;
         return ogc;
     },
-    getGmlPointElement: function(coordinates, srsName, version) {
```

Comment:
These methods were public, so they could be used elsewhere, removing them could break a lot of code